test/handler/test_embertemplatelint_handler.vader

```diff
@@ -46,6 +46,33 @@ Execute(The ember-template-lint handler should parse lines correctly):
   \ ],
   \ ale_linters#handlebars#embertemplatelint#Handle(347, input_lines)
 
+Execute(The ember-template-lint handler should handle template parsing error correctly):
+  let input_lines = split('{
+  \ "/ember-project/app/templates/application.hbs": [
+  \   {
+  \     "fatal": true,
+  \     "moduleId": "app/templates/application",
+  \     "message": "Parse error on line 5 ...",
+  \     "line": 1,
+  \     "column": 1,
```

Comment:
Line/column are not cleanly returned in data object by ember-template-lint when template cannot be parsed at all - I decided not to forcefully extract it from text message for now.